assets/scss/_custom.scss

```diff
@@ -530,6 +530,10 @@ body {
     border-left-color: #444;
   }
 
+  .alert-info:has(h4):has(ul) h4 {
+    float: initial !important;
```

Comment:
Can we use a more [specific](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) selector to avoid the need for `!important`?

/lgtm
but I'd prefer not to use `!important` here.